lib/main.dart

```diff
@@ -56,20 +62,13 @@ class HelpAPaw extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp.router(
-      theme: ThemeData(
```

Comment:
Why was the theme data changed?

Reply:
Firebase dependency changes. ui_localization package no longer exists.